contract/src/main/java/com/ritense/valtimo/contract/json/patch/JsonPatchBuilder.java

```diff
@@ -82,8 +84,30 @@ public JsonPatchBuilder copy(JsonPointer from, JsonPointer to) {
         return this;
     }
 
+    /** Adds a JsonNode value to a json at the specified location. */
+    public JsonPatchBuilder addJsonNodeValue(JsonNode destination, JsonPointer path, JsonNode value) {
```

Comment:
Is there a test for this?

Reply:
This code was moved and is tested extensively here: 
https://github.com/valtimo-platform/valtimo-backend-libraries/blob/main/process-document/src/test/kotlin/com/ritense/processdocument/resolver/DocumentJsonValueResolverTest.kt#L341-L462